lavalink/enums.py

```diff
@@ -101,11 +101,12 @@ class NodeState(enum.Enum):
 
 
 class PlayerState(enum.Enum):
-    CONNECTING = 0
-    READY = 1
-    NODE_BUSY = 2
-    RECONNECTING = 3
-    DISCONNECTING = 4
+    CREATED = 0
+    CONNECTING = 1
+    READY = 2
+    NODE_BUSY = 3
+    RECONNECTING = 4
+    DISCONNECTING = 5
```

Comment:
Possible breaking change and this is a micro release
```suggestion
    CREATED = 7
    CONNECTING = 0
    READY = 1
    NODE_BUSY = 2
    RECONNECTING = 3
    DISCONNECTING = 4
```